src/groupByKey.ts

```diff
@@ -2,6 +2,16 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(items:[], key: string): GroupsMap<object> {
+  const result: GroupsMap<object> = {};
```

Comment:
i think that GroupsMap should be with <T> generic like it was in condition. Also, key in parameters should be not a string type, but 'keyof T'